packages/dx-grid-core/src/index.js

```diff
@@ -71,6 +71,10 @@ export * from './plugins/table-column-visibility/helpers';
 export * from './plugins/column-chooser/computeds';
 export * from './plugins/column-chooser/reducers';
 
+export * from './plugins/tree-data-state/reducers';
```

Comment:
This reducer is equal to the row-detail-state one. Can we extract a util that performs such a calculation? Something like `addOrRemove(array, item)`.